### ISSUE TITLE
stream not playing smoothly when published over TCP using ffmpeg

### ISSUE THREAD

**Issue by GsandeepR** [iizukanao/node-rtsp-rtmp-server#106]
Hi,
I tried to publish the stream over RTSP (TCP transport) using ffmpeg.When I playback the rtmp url ,the stream is not playing smoothly.I see a lot of distortions in the playback.
ffmpeg -re -i ~/Videos/big_buck.mpg -c:v libx264 -preset medium -vb 2000k -c:a aac -ab 128k -ar 44100 -f rtsp  rtsp://192.168.27.32:8080/live/ffmpeg
![node](https://user-images.githubusercontent.com/16095573/33938818-d784cc3e-e02e-11e7-8be1-d699bdc95b98.png)

**Comment by iizukanao** [comment]
@GsandeepR What video player are you using?

**Comment by GsandeepR** [comment]
I tried to play with ffplay and VLC . with both players stream is not smooth

**Comment by iizukanao** [comment]
@GsandeepR Does your video contain B-frames? If so, please re-encode the video without B-frames and try again.

**Comment by GsandeepR** [comment]
Yes ,the video has B frames .I used the flag -bf 0  and it was playing fine .Thank you
